packages/rest/README.md

```diff
@@ -52,16 +52,37 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Template of the OpenAPI spec, which will be filled with `paths` from
+    // controllers.
+    openApiSpec: {
+      info: {
+        title: 'My Application',
+        version: '1.0.0',
+      },
+      servers: [{url: 'http://127.0.0.1:8080'}],
+    },
+
+    // Optional configuration for API explorer
+    apiExplorer: {
+      // URL for the hosted API Explorer UI
+      // default to https://loopback.io/api-explorer
+      explorerUrl: 'http://petstore.swagger.io',
```

Comment:
Can we use a shorter name `url`? The fact that this setting is controlling API Explorer is already encoded in the parent property name `apiExplorer`.